projects/hr-portal/src/hr-portal.ts

```diff
@@ -1,83 +1,195 @@
+              </a>
+              <a href="tel:${row.data.Phone}">
+                <igc-icon-button collection="hr-icons" name="tel" class="small" variant="flat"></igc-icon-button>
+              </a>
+              <a href="https://www.linkedin.com" target="_blank">
+                <igc-icon-button collection="hr-icons" name="linkedIn" class="small" variant="flat"></igc-icon-button>
+              </a>
+            </div>
+          `;
   }
 
+  public dateTemplate = (ctx: IgcCellTemplateContext) => {
+    let row = ctx.cell.row;
+    const date = new Date(row.data.HireDate);
+    const formattedDate = date.toISOString().split('T')[0]; 
+    return html`
+      ${formattedDate}
+    `;
+  }
+
+  render() {
+    return html`
+      <link rel="stylesheet" href="node_modules/igniteui-webcomponents-grids/grids/themes/light/fluent.css" />
+      <igc-tree-grid
+      <igc-tree-grid
+        id="treeGrid"
+        .data="${this.data}"
+        primary-key="ID"
+        child-data-key="Employees"
+        row-selection="multipleCascade"
+        allow-filtering="true"
+        filter-mode="excelStyleFilter"
+        class="gridStyle"
+      >
+        <igc-paginator per-page="20"></igc-paginator>
+
+        <igc-grid-toolbar>
+          <igc-grid-toolbar-title>HR Portal</igc-grid-toolbar-title>
+          <igc-grid-toolbar-actions>
+            <igc-grid-toolbar-hiding></igc-grid-toolbar-hiding>
+            <igc-grid-toolbar-pinning></igc-grid-toolbar-pinning>
+            <igc-grid-toolbar-exporter>
+              <span slot="excelText">Export</span>
+            </igc-grid-toolbar-exporter>
+            <igc-grid-toolbar-advanced-filtering></igc-grid-toolbar-advanced-filtering>
+          </igc-grid-toolbar-actions>
+        </igc-grid-toolbar>
+
+        <igc-column field="Name" width="300px" ?sortable="true" pinned="true" .bodyTemplate="${this.avatarTemplate}"></igc-column>
+        <igc-column field="JobTitle" header="Job Title" data-type="string" min-width="200px" sortable="true"></igc-column>
+        <igc-column field="Department" data-type="string" min-width="200px" sortable="true"></igc-column>
+
+        <igc-column field="Location" data-type="string" ?sortable="true" .bodyTemplate="${this.countryIconTemplate}"></igc-column>
+        <igc-column field="Contacts" data-type="string" min-width="200px" filterable="false" .bodyTemplate="${this.contactsTemplate}"></igc-column> 
 
+        <igc-column field="HireDate" header="Hire Date" data-type="date" min-width="100px" sortable="true" .bodyTemplate="${this.dateTemplate}"></igc-column>
+
+        <igc-column field="GrossSalary" header="Gross Salary" data-type="currency" min-width="100px" sortable="true"></igc-column></igc-column>
+      </igc-tree-grid>
+    `;
+  }
 
   static styles = css`
-    :host {
-      max-width: 1280px;
+  :host {
+      display: block;
       margin: 0 auto;
-      padding: 2rem;
+      align-items: center;
```

Comment:
If this is gonna be a block container, then you dont need `align-items`, `justify-content` and `gap`. Probably, wont need the `margin` and `text-allign` as well

Reply:
removed the unnecessary properties